Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -95,6 +95,24 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
           if (otherRowFailures.Count > 0)
             validationFailureRepository.AddValidationFailuresForDataRow(rowObject, otherRowFailures);
         }
+
+        var editModeController = ((IBocList)bocListControl).EditModeController;
+        if (editModeController.IsRowEditModeActive)
+        {
+          var bocListRow = editModeController.GetEditedRow();
+          var editableRow = editModeController.GetEditableRow(bocListRow.Index);
```

Comment:
```suggestion
          var editableRow = editModeController.GetEditedRow();
```
Unless I'm missing something in the API?

Reply:
`GetEditedRow` does not return an `IEditableRow` for some reason so this hop is required

Reply:
Ah, yes. no, that's actually correct and sensible after looking into the API. The naming might not be perfect and stems from the name on BocList itself. so, no change. GetEditedRow is just returning metadata.